server/controllers/entities/lib/validate_entity.ts

```diff
@@ -14,26 +16,31 @@ const allowlistedTypes = allLocallyEditedEntitiesTypes
 type ValidatableEntity = SetOptional<Pick<InvEntity, '_id' | 'labels' | 'claims'>, '_id'>
 
 // Can be used to validate both entities being created or existing entities
-export async function validateInvEntity (entity: ValidatableEntity) {
+export async function validateInvEntity (entity: ValidatableEntity, userIsAdmin?: boolean) {
   try {
-    return await validate(entity)
+    return await validate(entity, userIsAdmin)
   } catch (err) {
     if (err.context == null) err.context = { entity }
     throw err
   }
 }
 
-async function validate (entity: ValidatableEntity) {
-  if (isLocalEntityLayer(entity)) return
-
+async function validate (entity: ValidatableEntity, userIsAdmin?: boolean) {
   const { _id, labels, claims } = entity
   assert_.object(labels)
   assert_.object(claims)
-
-  const type = getValueType(claims)
-  validateValueType(type, claims['wdt:P31'] as WdEntityUri[])
-  validateLabels(labels, type)
-  return validateAndFormatInvClaims({ _id, type, claims })
+  const isLocalLayer = isLocalEntityLayer(entity)
+  let type
+  if (isLocalLayer) {
+    const remoteEntityUri = getFirstClaimValue(claims, 'invp:P1')
+    const remoteEntity = await getEntityByUri({ uri: remoteEntityUri })
+    type = remoteEntity.type
+  } else {
+    type = getValueType(claims)
+    validateValueType(type, claims['wdt:P31'] as WdEntityUri[])
+    validateLabels(labels, type)
```

Comment:
Is there a reason why local layer entities are now excluded from those validations: `validateValueType` and `validateLabels`? Admins could be mistaken too when they're typing.

Reply:
Because at this point, the local layer is only that: a local layer, without the remote entities labels and claims, in particular its `wdt:P31` claims